SwiftUIDZ/ActivityView.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  ActivityView.swift
+//  UIActivity&UIViewRepresentable
+//
+//  Created by 2lup on 18.01.2023.
+//
+
+import SwiftUI
+
+// Внедрение ActivityView в SwiftUI
+struct ActivityView: UIViewControllerRepresentable {
+    // MARK: - Typealias
+    typealias UIViewControllerType = UIActivityViewController
+    
+    // MARK: - Public Properties
+    
+    public var activityItems: [Any]
+    public var applicationActivities: [UIActivity]?
+    
+    // MARK: - Public Methods
+    
+    func makeUIViewController(context: Context) -> UIActivityViewController {
+        let vc = UIActivityViewController(activityItems: activityItems, applicationActivities: applicationActivities)
```

Comment:
```suggestion
         UIActivityViewController(activityItems: activityItems, applicationActivities: applicationActivities)
```
лишняя константа сразу возвращай